src/Umbraco.Web/umbraco.presentation/umbraco/translation/details.aspx.cs

```diff
@@ -88,7 +88,10 @@ public details()
             foreach (PropertyType pt in page.ContentType.PropertyTypes) {
                 pageRow = pageTable.NewRow();
                 pageRow[ui.Text("name")] = pt.Name;
-                pageRow[ui.Text("value")] = page.getProperty(pt.Alias).Value;
+                if (page.getProperty(pt.Alias) != null && page.getProperty(pt.Alias).Value != null)

```

Comment:
Thanks @tranquan. You're using `page.getProperty(pt.Alias)` 3 times here, it's better to then separate it out into a separate variable, can you update this?